fre/app/generate_time_averages/cdoTimeAverager.py

```diff
@@ -9,7 +9,17 @@ class inheriting from abstract base class timeAverager
     '''
 
     def generate_timavg(self, infile=None, outfile=None):
-        ''' use cdo package routines via python bindings '''
+        """
```

Comment:
I think we agreed to do the type hints as part of this doc update. Can you add those?